cirq/testing/circuit_compare.py

```diff
@@ -148,3 +163,37 @@ def assert_circuits_with_terminal_measurements_are_equivalent(
         'Diagram of reference circuit with desired function:\n'
         '{}\n'.format(actual, reference)
     )
+
+
+def assert_same_diagram(
```

Comment:
Rename to `assert_has_diagram`. I know that's the name I asked for, but it didn't occur to me that `same` is misleading since the two arguments have differing types.